src/eval.c

```diff
@@ -6130,11 +6132,11 @@ read_next_string_length(char_u *whole_string, int expect_literal_string, char_u
 }
 
 /*
- * Get a first "expr" of "${" "expr" "}" from `whole_embedding`.
+ * Get a first "expr" of "${" "expr" "}" from whole_embedding.
  *
- * whole_embedding starts with "${": e.g. "${10}`", "${var} Hello :D`".
+ * whole_embedding starts with "${": e.g. "${10}'", "${var} Hello :D'".
```

Comment:
Are the changes of this line really correct?
Shouldn't be `"${10}"` and `"${var} Hello :D"`?

Reply:
Yes, I intended this :D
e.g. `$'hello ${var} and ${10}'` passes `"${var} and ${10}'` and `"${10}'` to this function!